src/data2img.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import scipy.misc as smp
+import csv
+import os
+
+IMG_SIZE = 48
+csv_file_name = '../data/head.csv'
+data_output_dir = '../data/img'
+
+
+# based on https://stackoverflow.com/questions/312443/how-do-you-split-a-list-into-evenly-sized-chunks
+# based on https://stackoverflow.com/questions/434583/what-is-the-fastest-way-to-draw-an-image-from-discrete-pixel-values-in-python
+def csv2img(file_name):
```

Comment:
this should become some util method and moved somewhere else, as we are going to need it in some places #7